llvm/runtimes/CMakeLists.txt

```diff
@@ -171,12 +171,20 @@ if(compiler_rt_path)
   endif()
 endif()
 
+function(_get_runtime_name name out_var)
+  string(FIND ${name} "lib" idx)
+  if(idx EQUAL 0 AND NOT ${name} STREQUAL "libc")
+    string(SUBSTRING ${name} 3 -1 entry)
```

Comment:
In response to the issue seen by @JDevlieghere where the cxx and cxxabi targets are not present:

I believe ```entry``` needs changed to ```name``` for the SUBSTRING on line 177.

When using ```-DLLVM_ENABLE_RUNTIMES='libcxx;libcxxabi'```, ```${name}``` is being returned from the function instead of the SUBSTRING ```${entry}```. 

With your original patch the targets remain libcxx and libcxxabi.

Reply:
Thanks for catching that, I fixed it in fb619b3c78b1b6a026b0fec9fccff4c515465803.